cfg/aws/aws_sdk_logging.go

```diff
@@ -0,0 +1,61 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: MIT
+
+package aws
+
+import (
+	"log"
+	"strings"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/influxdata/wlog"
+)
+
+var sdkLogLevel aws.LogLevelType = aws.LogOff
+
+// SetSDKLogLevel sets the global log level which will be used in all AWS SDK calls.
+// Example usage: set `debug = true` and `aws_sdk_log_level = LogDebug` in config.json.
+func SetSDKLogLevel(wLogLevel wlog.Level, sdkLogLevelString string) {
+	if wLogLevel != wlog.DEBUG {
+		sdkLogLevel = aws.LogOff
+		return
+	}
+	var temp aws.LogLevelType = aws.LogOff
+	// Hard coded strings that match actual variable names in the AWS SDK.
+	// I don't expect these names to change, or their meaning to change.
```

Comment:
Hey now, no need to get defensive :-) 

It is very unlikely the AWS SDK would break backwards compatibility and change the names of the constants.
Even if it did, CloudWatch Agent would fail to compile.
So we would very quickly realize it.

If the AWS SDK adds new log levels we won't automatically know about them, but it won't be a regression in functionality. And we can add them if and when we want to use the new LogLevels.

If there were frequent additions to the AWS SDK LogLevels, then we would potentially need frequent updates to this code. I think that is very unlikely.

For what it's worth I don't thin Golang supports programmatically checking if there is a new constant `LogDebugWithFoo` because the constants are untyped:
https://stackoverflow.com/questions/45888678/how-to-get-all-constants-of-a-type-in-go

Reply:
Sounds reasonable to me. Would be nice if Golang supported enums more natively than defining named, untyped constants, but it is what it is.